blog/_posts/2017-08-18-gsoc-connecting-contributors-with-projects.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: blog-detail
+post-type: blog
+by: Michael Viveros
+title: GSOC - Connecting Contributors with Projects
+---
+
+## Introduction
+
+One problem with open-source software is that it can be difficult to find a project to contribute to and actually get started contributing to it. Some open-source projects can seem very intimidating to newcomers since they have a very large code base and minimal documentation about how to get involved with the project. Other projects are are better and have:
+- **a contributing guide** with specific steps people can follow to get involved
+- some well-documented, easy to fix, **beginner-friendly issues** which are great places to learn about the project and get started contributing
+- **a chatroom** like gitter where anyone can come and easily ask questions
+
+For Google Summer Of Code 2017, my [project](https://summerofcode.withgoogle.com/projects/#5367340028919808) was to make it easier for potential contributors to both find projects to contribute to and to get started contributing to projects, all through [Scaladex](https://index.scala-lang.org/) (the Scala Library Index). I accomplished this by highlighting projects that have Contributing Info (contributing guide, beginner-friendly issues and chatroom) on the [front page](https://index.scala-lang.org/) of Scaladex so that potential contributors can easily view projects that they could contribute to and see all the information necessary for contributing in one place. I also added a [Contributing Search page](https://index.scala-lang.org/search?q=&contributingSearch=true) which can be used to query these projects.
+
+[![front-page-contributing](/resources/img/blog/scaladex/front-page-contributing.png)](/resources/img/blog/scaladex/front-page-contributing.png)
+*Highlighted projects with Contributing Info on the front page of Scaladex*
+
+Furthermore, I improved the search feature of Scaladex by adding [Github Topics](https://github.com/blog/2309-introducing-topics) to the projects stored in Scaladex so that users can search projects based on Topics. Topics are essentially categories that open-source projects belong to like android, databases, json, ...
+
+[![front-page-topics](/resources/img/blog/scaladex/front-page-topics.png)](/resources/img/blog/scaladex/front-page-topics.png)
+*Topics for projects on the front page of Scaladex*
+
+## Contributing Info
+### How it Works
+As mentioned above, Contributing Info corresponds to 3 pieces of information, some of which are auto-populated when Scaladex is indexed, and all of which can be manually set by a project maintainer in their project's edit page in Scaladex:
```

Comment:
"As mentioned above, Contributing Info corresponds to 3 pieces of information, some of which are auto-populated when Scaladex is indexed, and all of which can be manually set by a project maintainer in their project's edit page in Scaladex:" -> "To Scaladex, _Contributing Info_ consists of the three pieces of information mentioned above: a contributor's guide, a collection of beginner-friendly issues, and a link to a gitter chatroom for the project where newcomers can come to easily ask questions. And perhaps most importantly; Scaladex can automatically obtain much of this info for your project! It's also possible for a project maintainer to automatically set this information on their project's edit page in Scaladex."

Reply:
Should I get rid of the [list](https://github.com/scala/scala-lang/pull/730/files#diff-1f4e36ab0cb7393e0fe4743078be6b6bR11) below describing how each piece of contributing info can be set? It doesn't fit well with what you added, maybe just adding a connecting sentence like "Here's how each piece of contributing info can be set:" will make it fit better. 

I liked that list since it tells maintainers specifically how each piece of contributing info is set/auto-populated.

Reply:
Nah, keep the list. It sometimes helps to be a bit repetitive in blog articles; people tend to skim 😉